code/aind_auto_train/curriculums/uncoupled_no_baiting_2p2rwdDelay159.py

```diff
@@ -24,7 +24,7 @@
 # Note this could be any string, not necessarily one of the Task enums
 curriculum_name = Task.C0B0
 curriculum_version = "2.2rwdDelay159"
```

Comment:
Do you think it's better to use a new curriculum version like 2.3?

Reply:
I'm debating between 2.2.1 and 2.3 actually. Compared to 2.1, 2.2 should have higher autowater and less likely rolling back, but the rwddelay curriculum in some way skipped this change. Maybe it is better to be 2.2.1 to be consistent with other curriculums?

Reply:
I reverted changes in original 2.2 and added a new 2.2.1 curriculum. It's called uncoupled_no_baiting_2p2p1rwdDelay159.py.